package.json

```diff
@@ -90,10 +90,11 @@
     "@nguniversal/builders": "^9.0.0",
     "@nguniversal/express-engine": "^9.0.0",
     "@nguniversal/module-map-ngfactory-loader": "^9.0.0-next.9",
-    "@nrwl/angular": "^9.1.2",
-    "@nrwl/jest": "^9.1.2",
-    "@nrwl/node": "^9.1.2",
-    "@nrwl/workspace": "^9.1.2",
+    "@nrwl/angular": "^9.0.4",
+    "@nrwl/jest": "^9.0.4",
+    "@nrwl/node": "^9.0.4",
+    "@nrwl/nx-plugin": "^9.1.0",
+    "@nrwl/workspace": "^9.0.4",
```

Comment:
I guess we can use the versions from the base branch here - I guess this diff is due to the older branch and rebasing